reconcile/openshift_base.py

```diff
@@ -270,9 +273,13 @@ def realize_data(dry_run, oc_map, ri,
                    to deploy to the same namespace
     :param wait_for_namespace: wait for namespace to exist before applying
     :param no_dry_run_skip_compare: when running without dry-run, skip compare
+    :param override_enable_deletion: override calculated enable_deletion value
+    :param recycle_pods: should pods be recycled if a dependency changed
     """
     actions = []
     enable_deletion = False if ri.has_error_registered() else True
+    if override_enable_deletion is not None:
+        enable_deletion = override_enable_deletion
```

Comment:
We should only override if we don't have any errors, correct?. If not, `enable_deletion` should always be `False`

Reply:
It is dangerous to override to `True` in case of errors.